plugins/inventory/nb_inventory.py

```diff
@@ -820,6 +868,31 @@ def get_region_for_site(site):
         # Dictionary of site id to region id
         self.sites_region_lookup = dict(map(get_region_for_site, sites))
 
+    # Note: depends on the result of refresh_sites_lookup for self.sites_with_prefixes
+    def refresh_prefixes(self):
+        # Pull prefixes with "active" status only
+        url = self.api_endpoint + "/api/ipam/prefixes?status=active"
```

Comment:
I'm not sure about only pulling prefixes that are in `active` state. I will have to check but I don't know how often we filter things based on `status` - I am guessing the behavior is to pull everything regardless of `status`

I think as a user I'd probably want all prefixes associated and then determine on my own what do to with ones with different statuses, rather than having this bit of code decide for me.